AcademicManagerV1/WebContent/html/NewPersonForm.jsp

```diff
@@ -0,0 +1,157 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+	pageEncoding="UTF-8"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<meta charset="utf-8">
+<title>Person Form Page</title>
+<link rel="stylesheet" type="text/css"
+	href="../css/FrontEnd_Homepage.css" media="screen">
+<link href="https://fonts.googleapis.com/css?family=Roboto"
+	rel="stylesheet">
+<link rel="stylesheet"
+	href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+	integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+	crossorigin="anonymous">
+</head>
+
+
+<body>
+	<%
+		String type = request.getParameter("type");
+	%>
```

Comment:
You don't need this, since you are accessing the query parameters via ```${param.type}```. Please remove the assignment.